mobile/src/utils/calculatePreviousDateAccordingToRange.ts

```diff
@@ -0,0 +1,5 @@
+export const calculatePreviousDateAccordingToRange = (dateRange: number) => {
+  const now = new Date();
+
+  return new Date(now.getTime() - dateRange * 24 * 60 * 60 * 1000);
```

Comment:
Nice vanilla code.

We're using dayjs do manipulate dates in app, prioritize something like:

```js
dayjs().subtract(7, 'day')
```

instead of add new things in utils folder.